api/api/src/test/scala/uk/ac/wellcome/platform/api/works/WorksErrorsTest.scala

```diff
@@ -7,31 +7,31 @@ class WorksErrorsTest extends ApiWorksTestBase {
       assertIsBadRequest(
         "/works?include=foo",
         description =
-          "include: 'foo' is not a valid value. Please choose one of: ['identifiers', 'items', 'subjects', 'genres', 'contributors', 'production', 'notes', 'collection', 'images']"
+          "include: 'foo' is not a valid value. Please choose one of: ['identifiers', 'items', 'subjects', 'genres', 'contributors', 'production', 'notes', 'collection', 'images', 'parts', 'partOf', 'precededBy', 'succeededBy']"
```

Comment:
bit of a bikeshed but these tests annoy me, I don't think they should break every time we add an include - do you reckon we should change them to (eg) just test whether the message starts with `include: 'foo' is not a valid value`?

Reply:
I think its a nice feature of an API documenting what the valid values are. We could change the tests to not check the actual values, using a regex or something, maybe one for a later date.